### ISSUE TITLE
[7.7.2] Server crashes with "Total world volume is larger than the max supported size

### ISSUE THREAD

**Issue by Disane87** [StrangeLoopGames/EcoIssues#9858]
This issue seems to be related to #9261 but it occurs now on my server:

```
--BEGIN DUMP--
Dump Time
10/25/2018 10:18:50

Exception
Exception: ArgumentException
Message:Total world volume of 1600 x * 838,860799609375 y * 1600 z = 2147483647 is larger than the max supported size of int.MaxValue (2147483647).
Source:Eco.Shared

System.ArgumentException: Total world volume of 1600 x * 838,860799609375 y * 1600 z = 2147483647 is larger than the max supported size of int.MaxValue (2147483647).
   at Eco.Shared.Math.WorldPosition3i.Initialize(Vector3i worldSize)
   at Eco.Shared.Voxel.ChunkGrid`1.AddChunk(TChunk chunk)
   at Eco.Shared.Voxel.ChunkGrid`1.GetOrAddChunk(Vector3i chunkPos)
   at Eco.World.WorldChunkGrid.SetBlock(Type blockType, Vector3i worldPos, Object[] args)
   at Eco.Gameplay.Objects.WorldObject.PlaceWorldObjectBlocks()
   at Eco.Gameplay.Objects.WorldObject.BeginInitialize()
   at Eco.Gameplay.Objects.WorldObjectManager.<>c__DisplayClass14_0.<Eco.Core.Plugins.Interfaces.IInitializablePlugin.Initialize>b__0(KeyValuePair`2 pair)
   at System.Threading.Tasks.Parallel.<>c__DisplayClass42_0`2.<PartitionerForEachWorker>b__1()
   at System.Threading.Tasks.Task.InnerInvokeWithArg(Task childTask)
   at System.Threading.Tasks.Task.<>c__DisplayClass176_0.<ExecuteSelfReplicating>b__0(Object )

--END DUMP--
```

World-size is 160x160

The world works for now 20 days and the exception occured today the first time (at our usual daily server boot at 5am). 

As I investigated, all backups since yesterday 5pm are corrupt and wont work.

Pam in the discord channel wanted me to post this here. If you need the corrupt worlds, I can send you them.

**Comment by Deantwo** [comment]
Sounds like an object got thrown too high into the air or something.

**Comment by Disane87** [comment]
yeah, some of my players reported, that some vehicles flown away and disappeared. As a hint for you: the players were digging at the edge of the map, between 1600 and 0

**Comment by D3nnis3n** [comment]
Both is afaik fixed in the next version, the bad physics on world borders as well as the server crashing when things are over the height limit, instead they get deleted.

**Comment by seccombew** [comment]
Fixed in 7.8+
